LICENSE

```diff
@@ -0,0 +1,12 @@
+Source code in this repository is covered by one of two licenses: (i) the GNU General Public License
+(GPL) v3.0 (ii) the Creative Commons BY-NC-ND v4.0. Creative Commons licensed documents are found in
+the docs directory. Other files are GPL licensed.
+
+GPL v3.0: https://github.com/bitwarden/contributing-docs/blob/master/LICENSE_GPL.txt
+
+Bitwarden License v1.0: https://github.com/bitwarden/server/blob/master/LICENSE_CC.txt
+
```

Comment:
Shouldn't this be linking tot he CC license?

Reply:
Oops, thank you, yes I missed that Bitwarden license reference and link. Fixed now